### DISCUSSION TITLE
10 shutters closes as they want, not as I want!

### DISCUSSION THREAD

**Discussion by frank8001** [SukramJ/aiohomematic#965] [Q&A]
### I agree to the following

- [X] I have read the [documentation](https://github.com/danielperna84/custom_homematic#custom_homematic)
- [X] I am aware of the latest [release notes](https://github.com/danielperna84/custom_homematic/releases)
- [X] The Backend (CCU/Homegear/...) is working as expected and connected devices are controllable by its own UI.
- [X] I am running the latest version of the custom_component (and Home Assistant)

### The problem

Hello!

I have 10 shutters. The automation should close them after sunset with different individual offsets iE sunset + 1:31:45. Mostly it is only one shutter (the first in the row), which closes correctly. The others do just nothing. Sometimes (but rare) they all do as they should do. 
HomematicIP (local) sometimes causes   ERRORS like "not reachable", but I guess this is not in directly associated with the shutter-problem.
There are no problems when I start the automation manually from the menu.
Same problem exists while opening, but not as often as while closing.

I use:
Home Assistant 2023.1.7
HomematicIP (local) /RaspberryMatic
Raspi 3b+
Rolladenaktoren HM-LC-Bl1PBU-FM

thanks in advance

Frank

### What version of HomematicIP (local) has the issue?

2,4,0

### What was the last working version of HomematicIP (local)?

no last working version

### What type of installation are you running?

Home Assistant OS

### What type of installation are you running for your homematic backend?

RaspberryMatic Standalone

### What hardware are you running for your system?

Pi3+

### Which config details do you use

- [ ] Username and password
- [ ] TLS
- [ ] callback data ([see](https://github.com/danielperna84/custom_homematic#callback_host-and-callback_port))

### Which interfaces do you use?

- [ ] Homematic IP
- [X] Homematic (Bidcos-RF)
- [ ] Groups (Heating-Group)
- [ ] BidCos-Wired (HM-Wired)

**Comment by SukramJ** [comment]
I guess you are able to control every **single** device as expected.
From what i have read in the past (i don't use shutters), it's not the best solution to send commands to 10 shutters at the same time (dutcycle).
I would have a look at the [homematic forum](https://homematic-forum.de/) how groups of shutters are controlled there.

Looks like you have a problem with this automation, but i see no issue with this integration.

**Reply by emufan** [reply]
> From what i have read in the past (i don't use shutters), it's not the best solution to send commands to 10 shutters at the same time (dutcycle).

Only for the sake of completeness. I open and close 16 HmIP-BROLL most of the times at the same time. Just HA group (all_shutter) of the single entities. Didn't face any problems with this. Beside the unavailability of sometimes 1 device every some weeks. But this would be there (tested a lot) even if I want to steer this one and not together with all the others.

**Comment by frank8001** [comment]
yes, dutycycles... but there is an offset, which is different for every shutter so that the commands, which are sent to the aktors, are sent at different times. That should cause no problem with dutycycle

**Reply by SukramJ** [reply]
You could active the debug logging (requires HA restart):

Add this to the configuration.yaml

```
logger:
  default: warn
  logs:
    hahomematic.xml_rpc_proxy: debug
```

After executing your automation you should see lines like:

`2023-02-16 19:08:02.284 DEBUG (MainThread) [hahomematic.xml_rpc_proxy] __ASYNC_REQUEST: ('putParamset', ('VCU1223813:4', 'VALUES', {'LEVEL_2': 1.0, 'LEVEL': 1.0}))`
or
`2023-02-16 19:08:02.285 DEBUG (MainThread) [hahomematic.xml_rpc_proxy] __ASYNC_REQUEST: ('setValue', ('VCU0000043:1', 'LEVEL', 1.0))`

Should be 10 commands with `setValue` if you only use bidcos devices.
